JSDemos/Demos/Toast/Stack/description.md

```diff
@@ -1 +1,23 @@
-Zdes' morzet byt' vasze opisanie
\ No newline at end of file
+The DevExtreme Toast components support stacking. Use the [notify(message, stack)](/Documentation/ApiReference/Common/Utils/ui/#notifymessage_stack) or [notify(options, stack)](/Documentation/ApiReference/Common/Utils/ui/#notifyoptions_stack) method to enable this feature.
+
+These methods use a **stack** object that has the following structure: *{position, direction}*.
+
+## Specify Position
+
+You can set the `position` field to a string (see *'alias'* tab in the demo) or an object (see *'coordinates'* tab in the demo). Note that if you use coordinates for the `position` field, you need to specify one vertical and one horizontal coordinate only. For example, if you specify *'top'*, you do not need to specify *' bottom'*.
```

Comment:
We want to change the 'alias' and 'coordinates' titles of the radio group buttons.
Here are the suggestions:
'fixed' - 'absolute'
'fixed point' - 'coordinates'

Reply:
I like 'coordinates'. For the other one, I would say 'alignment', because it lists possible alignment styles.

Reply:
I don't like 'alignment'. We have a title 'Position', and then there are tabs with 'alias' and 'coordinates'. If I change it to 'alignment' and 'coordinates', the word 'Position' doesn't fit. Besides, I think that 'alignment' is used usually with text or in the context of aligning blocks in CSS horizontally or vertically. In the demo, we don't align toasts, we position them.

What do you think about the following?
Position
- relative    - coordinates

Reply:
My issue with "relative" is that it's not necessarily opposite to "coordinates". Coordinates may be relative. 

One other thing I can suggest is "Position: predefined | coordinates". That would imply we offer a few "predefined" choices like "top-right" or "bottom-left" and we allow to specify exact coordinates. 

And if you don't like that, then I guess it's okay to go with "relative" and "coordinates". It's not a big deal.